kevm

```diff
@@ -96,36 +96,36 @@ run_interpreter() {
 }
 
 run_test() {
-    local backend=$1        ; shift
-    local test_file="$1"    ; shift
+    local backend test_file output_file expected_file
+
+    backend=$1     ; shift
+    test_file="$1" ; shift
+
+    expected_file="tests/templates/output-success.json"
+    output_file="$(mktemp "$test_file.out.XXXXXX")"
+    trap "rm -rf $output_file" INT TERM EXIT
+
     case "$test_file" in
         *proofs/*)
             run_proof "$test_file" --debug "$@"
             ;;
         *interactive/*)
-            local expected_file="$1" ; shift
```

Comment:
Don't we still need this?

Reply:
No, actually we don't. I just hardcode it above now.

Reply:
I should update the `Makefile` though, 1 sec.

It's only used by `test-interactive`, not by any of the other ones. And even in that case it is hardcoded to `tests/templates/output-success.json`

Reply:
Ah crap, we do need it :/ I'll push a fix.

Reply:
@15joeybloom does it look correct now?

Reply:
lgtm!